firmware/config/boards/cypress/board.mk

```diff
@@ -20,5 +20,15 @@ BOARDINC += $(PDL_DIR)/driver $(PDL_DIR)/driver/gpio $(PDL_DIR)/driver/usb $(PDL
 BOARDCPPSRC += $(CONTROLLERS_ALGO_AUTO_GENERATED_ENUMS)
 
 # Define linker script file here
-LDSCRIPT= $(STARTUPLD_CYPRESS)/cypress_S6E2CxAH.ld
-
+LDSCRIPT = $(STARTUPLD_CYPRESS)/cypress_S6E2CxAH.ld
+
+# TODO: somehow this -DDUMMY is helping us to not mess up the parameters, why?!
+# https://github.com/rusefi/rusefi/issues/684
+DDEFS += -DDUMMY
```

Comment:
I bet that "DUMMY" was specifically about "export EXTRA_PARAMS" magic, if we go live without export EXTRA_PARAMS we shall try living without DUMMY and the TODO?

Reply:
I'm out of the loop re: this, but it also builds fine w/o `DUMMY`; I can push the commit here if you want, otherwise here's a PR w/ an added commit: https://github.com/rusefi/rusefi/pull/4753

Reply:
just do it here, no need for extra PR

Reply:
done; builds fine without `DUMMY` def.